schemas/stsci.edu/asdf/transform/remap_axes-1.3.0.yaml

```diff
@@ -0,0 +1,91 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/transform/remap_axes-1.3.0"
+tag: "tag:stsci.edu:asdf/transform/remap_axes-1.3.0"
+title: >
+  Reorder, add and drop axes.
+
+description: |
+  This transform allows the order of the input axes to be shuffled and
+  returned as the output axes.
+
+  It is a list made up of integers or "constant markers".  Each item
+  in the list corresponds to an output axis.  For each item:
+
+  - If an integer, it is the index of the input axis to send to the
+    output axis.
+
+  - If a constant, it must be a single item which is a constant value
+    to send to the output axis.
+
+  If only a list is provided, the number of input axes is
+  automatically determined from the maximum index in the list.  If an
+  object with `mapping` and `n_inputs` properties is provided, the
+  number of input axes is explicitly set by the `n_inputs` value.
+
+  Invertibility: This transform does not have a general analytical inverse.
+  In some well defined cases it is possible to invert automatically
+
+examples:
+  -
+    - For 2 input axes, swap the axes
+    - |
+        !transform/remap_axes-1.3.0
+          mapping: [1, 0]
+  -
+    - For 2 input axes, return the second axis and drop the first
+    - |
+        !transform/remap_axes-1.3.0
+          mapping: [1]
+
+  -
+    - For 2 input axes, return the first axis twice, followed by the second
+    - |
+        !transform/remap_axes-1.3.0
+          mapping: [0, 0, 1]
+
+  -
+    - For 2 input axes, add a third axis which is a constant
+    - |
+        !transform/remap_axes-1.3.0
+          mapping: [0, 1, 42]
+
+  -
+    - |
```

Comment:
Is it still valuable to show an example of fixing a constant?  If so, we should keep the transform/constant that was removed (and change the description, since this example is no longer equivalent to one above).  If not, this example can be removed.

Reply:
Yes, this needs to be removed